liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -804,8 +805,11 @@ protected void doExecute() throws Exception {
         util.startServer();
 
         // collect artifacts canonical paths in order to build classpath
-        List<String> compileArtifactPaths = project.getCompileClasspathElements(); 
-        List<String> testArtifactPaths = project.getTestClasspathElements();
+        List<String> compileArtifactPaths = new ArrayList<String>();
+        compileArtifactPaths.addAll(project.getCompileClasspathElements());
```

Comment:
Should this check if `project.getCompileClasspathElements()` is null before adding it to the list?